draft-ietf-quic-transport.md

```diff
@@ -7334,6 +7334,36 @@ change controller of the IETF and a contact of the QUIC working group
 (quic@ietf.org).
 
 
+## QUIC Versions Registry {#iana-version}
+
+IANA \[SHALL add/has added] a registry for "QUIC Versions" under a "QUIC"
+heading.
+
+The "QUIC Versions" registry governs a 32-bit space; see {{versions}}. This
+registry follows the registration policy from {{iana-policy}}. Permanent
+registrations in this registry are assigned using the Specification Required
+policy ({{!RFC8126}}).
+
+All fields in {{iana-provisional}} MUST be included in all registrations for
+this registry.
+
+The codepoint of 0x00000001 to the protocol is assigned with permanent status
+to the protocol defined in this document. The codepoint of 0x00000000 is
+permanently reserved; the note for this codepoint \[shall] indicate\[s] that
+this version is reserved for Version Negotiation. All codepoints that follow
+the pattern 0x?a?a?a?a are permanently reserved.
+
+\[\[RFC editor: please remove the following note before publication.]]
+
+IANA note:
+
+: Several pre-standardization versions will likely be in use at the time of
+  publication. There is no need to document these in an RFC, but recording
+  information about these version will ensure that the information in the
+  registry is accurate.  The document editors or working group chairs can
+  facilitate getting the necessary information.
```

Comment:
Given this note will be removed, you could easy note these versions here. This would be others (than the editor) the chance to review and make sure all version that are currently deployed are covered.

Reply:
Or simply link the wiki.

Reply:
Since this document will have a date, it might make more sense to include the list here.

Reply:
I would prefer to leave the dynamic information to the registry.  Documenting the mess of proprietary versions we currently know about isn't really something that is worth creating a permanent record of.

Reply:
Yeah, that's a fair point.